pages/game_page.py

```diff
@@ -1,3 +1,48 @@
 import streamlit as st
+import random 
 
-st.title("GAME PAGE")
\ No newline at end of file
+st.title("Build a Scientist")
+
+game_words = ['Genetic','Histology','Sequence','Mutation','Genome','Oncology']
+
+
+if game_words:
+    selected_word = random.choice(game_words)
+
+# Session state
+if 'selected_word' not in st.session_state:
+    st.session_state.selected_word = selected_word
+    
+if 'good_guesses' not in st.session_state:
+    st.session_state.good_guesses = []
+
+if 'bad_guesses' not in st.session_state:
+    st.session_state.bad_guesses = []
+
+# Display and update the game word as player enters new guesses 
+def game_word_generator(word):
+    st.write(word)
+    # Create a display string with underscores for each letter in the word
+    if not st.session_state.good_guesses:
+        final_word = st.write(f"`{' '.join(['_'] * len(word))}`")
+    else:
+        final_word = ' '.join([guess if guess in st.session_state.good_guesses else '_' for guess in word])
```

Comment:
Nice!! You could simplify this further by getting rid of the if else block above and just keeping `final_word = ' '.join([guess if guess in st.session_state.good_guesses else '_' for guess in word])` because if `st.session_state.good_guesses = []` then `_` will get written for every character in the word